crates/driver/src/domain/liquidity/zeroex.rs

```diff
@@ -1,5 +1,68 @@
-/// A signed 0x Protocol Limit Order [^1].
-///
-/// [^1]: <https://docs.0x.org/limit-orders-advanced-traders/docs/introduction>
+use {
+    crate::domain::{eth, liquidity},
+    anyhow::anyhow,
+    contracts::IZeroEx,
+    ethcontract::Bytes,
+    primitive_types::{H160, H256, U256},
+};
+
 #[derive(Clone, Debug)]
-pub struct LimitOrder {}
+pub struct LimitOrder {
+    pub order: Order,
+    pub zeroex: IZeroEx,
```

Comment:
I believe other liquidity implementations would have the `IZeroEx` struct via an `Arc` because it's pretty big and always identical.